R/event_sources.R

```diff
@@ -0,0 +1,272 @@
+#' Pre-Defined Event Objects
+#'
+#' These pre-defined `event()` and `event_joined()` objects can be used as input
+#' to `admiral::derive_extreme_event()`.
+#'
+#' @details
+#' To see the definition of the various objects simply print the object in the
+#' R console, e.g. `bor_sd`. For details of how to use these objects
+#' please refer to [admiral::derive_extreme_event()].
+#'
+#' It is assumed that `dataset_name = "ovr"` refers to the dataset of the
+#' overall response assessments which should be considered for the parameter
+#' derivations. For example the dataset should not include assessment after PD
+#' or start of anti-cancer therapy.
+#'
+#' @seealso [admiral::derive_extreme_event()], [admiral::event()], [admiral::event_joined()]
+#'
+#' @format NULL
+#'
+#' @export
+#'
+#' @keywords source_specifications
+#'
+#' @rdname event_objects
+#'
+#' @examples
+#' # This shows the definition of all pre-defined `event` objects that ship
+#' # with {admiralonco}
+#' exports <- sort(getNamespaceExports("admiralonco"))
+#' for (obj_name in exports) {
+#'   obj <- getExportedValue("admiralonco", obj_name)
+#'   if (inherits(obj, "event_def")) {
+#'     cat("\n", obj_name, ":\n", sep = "")
+#'     print(obj, indent = 2)
+#'   }
+#' }
+rsp_y <- event(
+  description = "Define CR or PR as (unconfirmed) response",
```

Comment:
i really like this description addition for users benefit - should we add it for each of the `tte_sources` too?

Reply:
Maybe, I have added it as point to the [clean-up discussion](https://github.com/pharmaverse/admiral/discussions/2010).